uiowa/uiowa
# Hotfix - Null check on region items before block render

## To Test

- Create a region item and assign it to the region pre-footer.
- Delete region item entity.
- Page should not error out.
